components/SensorComponent.js

```diff
@@ -20,77 +15,128 @@ const isShaking = (data) => {
 const ShakeView = styled.View`
   display: flex;
   flex-direction: column;
+  align-items: center;
 `;
 
-const ShakeAlert = styled.Text`
+const Wrapper = styled.View`
+  background-color: white;
+  margin-top: 30%;
+  align-items: center;
+`
+
+const Header = styled.Text`
   font-size: 36px;
   font-weight: bold;
-  color: #aa0000;
+  color: #f54284;
+  margin-top: 20px;
 `;
-const ShakeDataView = styled.View``;
-const ShakeDataTitle = styled.Text`
+
+const Shake = styled.Text`
+  font-size: 20px;
   font-weight: bold;
+  color: pink;
+  margin-bottom: 20px;
+`;
+
+const ImageContainer = styled.View`
+  margin: 5%;
 `;
-const ShakeData = styled.Text``;
+
+const StyledButton = styled.TouchableOpacity`
+  margin: 10px;
+  color: pink;
+`
+
+// ==========================
+// = Function
+  //När ny data kommer in, kallas Accelerometer-eventet. Frekvensen är kontrollerad av setUpdateInterval(millisekunder).
+  //Subscribe = för att sluta använda Accelerometer(oklart). Tar man bort unsubscribe blir det segt och drar mycket batteri.
+  //useEffecten lyssnar när SensorComponent är aktiv.
```

Comment:
I liked your explanation here. So for me it was great. 
If someone non-swedish-speaking-person should look at your code they might be confused. :)